src/psyclone/psyir/nodes/whileloop.py

```diff
@@ -0,0 +1,183 @@
+# -----------------------------------------------------------------------------
```

Comment:
Please rename this file to be `while_loop.py`.

Reply:
Done.